scipy/_lib/_util.py

```diff
@@ -16,18 +16,6 @@ class Generator():  # type: ignore[no-redef]
         pass
 
 
-def _valarray(shape, value=np.nan, typecode=None):
```

Comment:
This is double underscored `_util` and `_valarray` but should we possible alias it to `np.full` for one release with a runtime warning?